src/fetcher.ts

```diff
@@ -45,12 +58,29 @@ async function fetchIDL(source: IDLSource) {
             }
             return !previous.classList.contains("atrisk") && !previous.textContent!.includes("IDL Index");
         });
-    if (!elements.length) {
-        throw new Error(`Found no IDL code from ${source.url}`);
+    return elements.map(element => trimCommonIndentation(element.textContent!).trim()).join('\n\n');
+}
+
+function extractCSSDefinitions(dom: DocumentFragment) {
+    const properties =  Array.from(dom.querySelectorAll("dfn.css[data-dfn-type=property],.propdef dfn"))
```

Comment:
double-space here

Reply:
You should probably create cssSelector to match idlSelector instead of having a literal here.